### PR TITLE
feat(capacity_parsers):Change to openelectricity 

### PR DESCRIPTION
**Pull request by annaib00**
## Issue
Changed data source and null updates

## Description
The source has changed from opennem to openelectricity, this change is now implemented here. The file name has also been changed and the dependencies in the dependent AU yaml files. 

Then there has been an issue of updating the capacities with null values, this is fixed by implementing a dropna statement. I would like a second opinion on if this is sufficient. 

### Preview
image

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: electricitymap/contrib/capacity_parsers/OPENELECTRICITY.py [hunk 1]
```diff
@@ -97,6 +112,8 @@ def fetch_production_capacity_for_all_zones(
         capacity_df.groupby(["zone_key", "mode"])[["value"]].sum().reset_index()
     )
 
+    capacity_df = capacity_df.dropna(subset=["value"])
```
Comment (annaib00) [inline electricitymap/contrib/capacity_parsers/OPENELECTRICITY.py:112]: I would like a second opinion on whether this is the good way to not get null capacity updates?

File: electricitymap/contrib/capacity_parsers/tests/test_OPENELECTRICITY.py
Comment (annaib00) [inline electricitymap/contrib/capacity_parsers/tests/test_OPENELECTRICITY.py:1]: I have also updated the capacity test to fit with the new file name

File: electricitymap/contrib/capacity_parsers/OPENELECTRICITY.py (hunk 1 above)
Comment (VIKTORVAV99) [reply electricitymap/contrib/capacity_parsers/OPENELECTRICITY.py:112]: Should be but maybe we can add a snapshot test with some deliberate null values in the mock data to ensure it works as expected?

Comment (annaib00) [reply electricitymap/contrib/capacity_parsers/OPENELECTRICITY.py:112]: That is a good idea, however I am not entirely sure how to implement this?

Comment (VIKTORVAV99) [reply electricitymap/contrib/capacity_parsers/OPENELECTRICITY.py:112]: I'd intercept the http call with a mock function and then just check what the parser returns.

Looks like this:
```
def test_fetch_consumption(adapter, session, snapshot):
    data = Path(base_path_to_mock, "DK-DK1_consumption.xml")
    adapter.register_uri(
        GET,
        ANY,
        content=data.read_bytes(),
    )

    assert snapshot == ENTSOE.fetch_consumption(ZoneKey("DK-DK1"), session)
```

Comment (VIKTORVAV99) [reply electricitymap/contrib/capacity_parsers/OPENELECTRICITY.py:112]: Might have to use read_text() instead tho as this returns JSON

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

**Comment by VIKTORVAV99** [comment]
Let's hope it works now!
